### DISCUSSION TITLE
laravel 11

### DISCUSSION THREAD

**Discussion by Malebestia** [konnco/filament-import#155] [Q&A]
Hi, Laravel has been updated, how long do you think it will take to get the updated package for Laravel 11?

**Comment by dade1987** [comment]
> Hi, Laravel has been updated, how long do you think it will take to get the updated package for Laravel 11?

I need it too

**Comment by uqjohart** [comment]
Any luck getting it on larvael 11? I'd hate to see this awesome plugin die :c
